pkg/costmodel/router.go

```diff
@@ -1398,6 +1398,38 @@ func (a *Accesses) Status(w http.ResponseWriter, r *http.Request, _ httprouter.P
 	}
 }
 
+type LogLevelResponse struct {
+	Level string `json:"level"`
+}
+
+func (a *Accesses) GetLogLevel(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+
+	level := log.GetLogLevel()
+	llr := LogLevelResponse{
+		Level: level,
+	}
+
+	body, err := json.Marshal(llr)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("unable to retrive log level"), http.StatusInternalServerError)
+		return
+	}
+	w.Write(body)
+}
+
+func (a *Accesses) SetLogLevel(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	qp := httputil.NewQueryParams(r.URL.Query())
+	level := qp.Get("level", "")
+	err := log.SetLogLevel(level)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("level must be a valid log level according to zerolog; level given: %s", level), http.StatusBadRequest)
+		return
+	}
```

Comment:
Great error message. Would you also include `err` somewhere in the message, just in case its something unexpected?